### PR TITLE
Document with contributors

### PR DESCRIPTION
**Pull request by jakelishman**

### CODE REVIEW

File: contributors.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Luciano Bello
+Matthew Treinish
+Michael Healy
+Noah Wood
+Pranav Gokhale
+Prasahnt Sivarajah
+Salvadore de la Puente
+Thomas Alexander
+Travis Scholten
+Yunong Shikj
```
Comment (k4rtik) [inline contributors.md:37]: @jakelishman I believe Yunong's last name is "Shi".

Comment (jakelishman) [reply contributors.md:37]: oh whoops thanks - that's what I originally typed, then fidgeted around in vim without exiting insert mode!

**Review by blakejohnson — DISMISSED** [review]
What did you pick as a cutoff date for gather contributors by commits? I believe your list corresponds to people that have made a commit in the OpenQASM 3 "era".

**Comment by jakelishman** [comment]
I ran some variant of `git shortlog --no-merges 2.0...` - i.e. everything on `main` after the commit tagged `2.0`, and then very mildly filtered to avoid double-counting people who didn't have consistent names in their commits, or who were already counted for other contributions.  I used that tag, rather than an explicit date.

**Comment by jakelishman** [comment]
I have subsumed this into #413, so it's all in the same place for posterity for the election.  There was never any particular intent to merge this PR - it was just a convenient way of receiving comments.

**Pull request closed without merging**